dspace-api/src/test/java/org/dspace/builder/ItemBuilder.java

```diff
@@ -364,6 +364,11 @@ public ItemBuilder withAdminUser(EPerson ePerson) throws SQLException, Authorize
         return setAdminPermission(item, ePerson, null);
     }
 
+    public ItemBuilder withSubmitter(EPerson submitter) {
+        item.setSubmitter(submitter);
+        return this;
+    }
```

Comment:
this must be done setting the submitter as currentUser in the context otherwise many internal service will behave differently than in real code

Reply:
(We will be reverting, so no further changes)